executor/oci/spec_windows.go

```diff
@@ -119,3 +119,9 @@ func generateCDIOpts(_ *cdidevices.Manager, devices []*pb.CDIDevice) ([]oci.Spec
 	// https://github.com/cncf-tags/container-device-interface/issues/28
 	return nil, errors.New("no support for CDI on Windows")
 }
+
+func getMountType(_ string) string {
```

Comment:
Isn't this true only for bind? In that case should be `normalizeBindType()` with an input parameter comparison.

Reply:
Not just for bind, all of them are referred to as `windows-layer` by this point. They all fail with:
```
failed to create shim task: invalid OCI spec - Type 'windows-layer' not supported: unknown
```
However, I think I can rename this to `normalizeMountType` instead?